packages/manager/src/features/Databases/DatabaseDetail/DatabaseSummary/DatabaseSummary.tsx

```diff
@@ -11,23 +24,24 @@ interface Props {
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const DatabaseSummary: React.FC<Props> = (props) => {
+  const classes = useStyles();
   // const databases = useDatabases();
   // const { databaseID } = props;
   // const thisDatabase = databases.databases.itemsById[databaseID];
 
   return (
-    <>
-      Database Summary
-      {/* <DatabaseSummaryLabelPanel
-        databaseID={thisDatabase.id}
-        databaseLabel={thisDatabase.label}
-      />
-      <DatabaseSummaryPasswordPanel databaseID={thisDatabase.id} />
-      <DatabaseSummaryMaintenancePanel
-        databaseID={thisDatabase.id}
-        databaseMaintenanceSchedule={thisDatabase.maintenance_schedule}
-      /> */}
-    </>
+    <Paper>
+      <Grid container>
+        <Grid item>
+          <ConnectionDetails /> 
+        </Grid>
+        <Grid item>
+          <ClusterConfiguration />
+        </Grid>
+      </Grid>
```

Comment:
Switch the order of these to match the spec and remove trailing space

```suggestion
        <Grid item>
          <ClusterConfiguration />
        </Grid>
        <Grid item>
          <ConnectionDetails />
        </Grid>
      </Grid>
```